### ISSUE TITLE
[ERROR] admin_general#index (I18n::InvalidLocale) ":en is not a valid locale"

### ISSUE THREAD

**Issue by garethrees** [mysociety/belgium-theme#57]
```
An I18n::InvalidLocale occurred in admin_general#index:

:en is not a valid locale
app/views/admin_general/index.html.erb:171:in `_app_views_admin_general_index_html_erb___73758833132860574_68625400'


-------------------------------
Request:
-------------------------------

* URL : https://transparencia.be/fr_BE/admin
* HTTP Method: GET
* IP address : FILTERED
* Parameters : {"controller"=>"admin_general", "action"=>"index", "locale"=>"fr_BE"}
* Timestamp : 2017-07-15 18:13:13 +0200
* Server : FILTERED
* Rails root : /FILTERED/belgium.alaveteli.org/alaveteli-2017-06-12T10-25-30
* Process: 22203

-------------------------------
Backtrace:
-------------------------------

app/views/admin_general/index.html.erb:171:in `_app_views_admin_general_index_html_erb___73758833132860574_68625400'
app/controllers/application_controller.rb:108:in `record_memory'
lib/strip_empty_sessions.rb:13:in `call'
```

**Comment by lizconlan** [comment]
Should be fixed by https://github.com/mysociety/alaveteli/pull/4073

**Comment by lizconlan** [comment]
Oh, unless it's to do with stored default locales being stale. I've assigned this to myself to check after the fix is deployed

**Comment by lizconlan** [comment]
Error is triggered by the call to `InfoRequest::OLD_AGE_IN_DAYS` which is calling `21.days`

and hitting this:

https://github.com/rails/rails/blob/4-0-stable/activesupport/lib/active_support/duration.rb#L72-L80
(explicit call to `parts.to_sentence(:locale => :en)`)

(This is not from our call to `.inspect` in the template code, just calling `21.days` in the console triggers the error)

**Comment by lizconlan** [comment]
Looks like it's fixed in Rails 4.1, so this is a Rails 4.0 bug

https://github.com/rails/rails/blob/4-1-stable/activesupport/lib/active_support/duration.rb#L76-L82

**Comment by lizconlan** [comment]
Closed via #4164
